### ISSUE TITLE
Can't locate utilities colors 

### ISSUE THREAD

**Issue by iamsukri** [twbs/bootstrap#28602]
I'm trying to set my text using default text-primary and the others but there's no change. I cannot locate the utilities colors. I'm using scss for bootstrap and would like to change the variables rather than creating text-primary classes which already be created based on the documentations.

Anyone have any idea where is text-primary/success etc. located?

I'm using the latest version of bootstrap 4.3.1

**Comment by Lausselloic** [comment]
Yes it's in `scss/_variables.scss`https://github.com/twbs/bootstrap/blob/efb9a7749f2d270050bd935492ee9992ceb43714/scss/_variables.scss#L69

**Comment by iamsukri** [comment]
hey. I just figured. 

I actually made a duplicate copy of the variables.scss. and only modify variables that I need and commented out the rest so it refers to original variables.scss.

**adding/modifying colors**
`$theme-colors: (
"primary": $brand-primary
);`

I modify variable-"primary" in my duplicate copy and have the other default colours to refer to original variables.scss. Once compiled I realised that "success/info/warning/dark and the others" are not able to refer to the original variables.scss default colours.

I solved it by including the other default variables into my duplicated copy.

I assumed that the compiler didn't include default $theme-colors from the original variable.scss and only execute my duplicate variables.scss
